packages/webview_flutter/webview_flutter_wkwebview/lib/src/ui_kit/ui_kit.dart

```diff
@@ -12,6 +12,10 @@ import '../foundation/foundation.dart';
 import '../web_kit/web_kit.dart';
 import 'ui_kit_api_impls.dart';
 
+// TODO(bparrishMines): All subclasses of NSObject need to pass their
+// InstanceManager and BinaryMessenger to its parent. They also need to
+// override copy();
```

Comment:
Should this have a tracking issue?

Reply:
done: https://github.com/flutter/flutter/issues/105245